blog/_posts/2017-08-10-opensource-here-we-come.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Open source, here we come!
```

Comment:
Hi, Thanks for your PR. :) Few minor changes are needed. Check out the comments below.